File: 349-intersection-of-two-arrays/intersection-of-two-arrays.py
class Solution:
    def intersection(self, nums1: List[int], nums2: List[int]) -> List[int]:
        res_nums=[]
        for i in nums1:
            if i in nums2:
                if i not in res_nums:
                    res_nums.append(i)
                    
        return res_nums

File: 349-intersection-of-two-arrays/intersection-of-two-arrays.java
class Solution {
    public int[] intersection(int[] nums1, int[] nums2) {
        List<Integer> nums3 = new ArrayList<>();

        for(int i =0;i<nums1.length;i++){
            for (int j =0;j<nums2.length;j++){
                if(nums1[i] == nums2[j]){
                    if (!nums3.contains(nums1[i])) {
                        nums3.add(nums1[i]);
                    }
                }
            }
        }
        int[] result = new int[nums3.size()]; // Create an int[] with the correct size
        for (int i = 0; i < nums3.size(); i++) {
            result[i] = nums3.get(i);
            }
            return result; 
    }
}